pkg/virt-config/config-map.go

```diff
@@ -248,6 +254,8 @@ func setConfig(config *Config, configMap *k8sv1.ConfigMap) error {
 		config.MachineType = machineType
 	}
 
+	config.CPUModel = strings.TrimSpace(configMap.Data[cpuModelKey])
```

Comment:
I guess that `CPUModel` has a default of `""`, still could you follow the pattern from above, but it is not clear now how you want to interpret it when it it is missing. Could you clarify that in the code?

See all examples above, they explicitly treat `""` as "keep the default". So a missing field or an empty string have the same meaning. It is also possible to differentiate between `""` and a not existing key if you need to.

Reply:
> I guess that `CPUModel` has a default of `""`, still could you follow the pattern from above

No problem, it is equivalent - changed.

> , but it is not clear now how you want to interpret it when it it is missing. Could you clarify that in the code?
> See all examples above, they explicitly treat `""` as "keep the default". So a missing field or an empty string have the same meaning. It is also possible to differentiate between `""` and a not existing key if you need to.

I want both cases to be the same: no default CPU model should be set in those cases. Therefore the empty string is returned and is ignored in mutating-webhook. Could you please be more concrete about what needs to be clarified in the code and where?

Reply:
You did the change I wanted. :+1: Basically keeping the default if the string is `""` instead of setting it to `""`. The result is the same, because the default is `""` but it is explicit now.